_treehouses

```diff
@@ -255,6 +256,22 @@ treehouses shadowsocks restart
 treehouses shadowsocks remove
 treehouses shadowsocks start
 treehouses shadowsocks stop
+treehouses storage
```

Comment:
`treehouses storage` doesn't do anything but return the help command, so this isn't needed here.